### PR TITLE
fix css for discord page

### PR DESCRIPTION
**Pull request by ashishmangla0**
before:
image

after: 
image

### CODE REVIEW

File: discord.html [hunk 1]
```diff
@@ -1,200 +1,186 @@
...
+                    </li>
+                  </ol>
+                </li>
 
-                  <li class="category-name">
-                    <strong>Text:</strong> Contains multiple other channels in
-                    it.
-                  </li>
+                <li class="category-name">
+                  <strong>Text:</strong> Contains multiple other channels in
```
Comment (Rucha1499) [inline discord.html:152]: Can we change `Text` to `Text Channels` like we have it in our discord server ?

**Review by Rucha1499 — CHANGES_REQUESTED** [review]
Great work!! @ashishmangla0 🎉 
The discord page looks good now! ⭐ 

Adding some review comments. kindly check. Thank you!

File: discord.html (hunk 1 above)
Comment (ashishmangla0) [reply discord.html:152]: fixed

File: discord.html [hunk 2]
```diff
@@ -1,200 +1,186 @@
...
-                  </li>
+                <li class="">
+                  <strong>Main: </strong> It has channels for general
+                  disussion,progress-updates ,newcomers & Support.
+                  <ol class="channel-info">
+                    <li class="channel-name">
+                      general:It carries general discussions.
+                    </li>
+                    <li class="channel-name">
+                      newcomers:All info related to new commers are floated in
```
[the remark this answers is not in the document]
Comment (ashishmangla0) [reply discord.html]: fixed

File: discord.html (hunk 1 above)
Comment (Rucha1499) [reply discord.html:152]: I think you changed it at different place. 😅

File: discord.html [hunk 3]
```diff
@@ -1,200 +1,186 @@
...
+                  different projects.
+                </li>
 
-                  <li class="category-name">
-                    <strong>Tasks:</strong> Channels for seperate tasks.
-                  </li>
-                </ul>
-              </div>
+                <li class="category-name">
+                  <strong>Text Channels : </strong> Channels for seperate tasks.
```
Comment (Rucha1499) [inline discord.html:167]: https://github.com/Real-Dev-Squad/website-welcome/pull/150#discussion_r601620963

Here it is `Tasks`!

Comment (ashishmangla0) [reply discord.html]: done

File: discord.html (hunk 1 above)
Comment (ashishmangla0) [reply discord.html]: done

**Review by Rucha1499 — APPROVED** [review]
Looks great! 🔥  Thank you.

**Pull request merged**